getting_started.md

```diff
@@ -31,17 +36,26 @@ can be installed, the repository needs to be added to the IDE.
    button
 1. Once downloaded, click the **Restart IntelliJ IDEA** button
 
-### Set up the Flutter SDK
+### Flutter plugin configuration
 
-See [flutter.io/setup](https://flutter.io/setup/) for instructions about installing the Flutter SDK
-and configuring your machine for development.
+1. Open plugin preferences (**Preferences>Plugins**)
+1. Select **Languages & Frameworks>Flutter**
+1. Set the path to the root of your flutter repo
+1. Click OK
 
 ### Creating a project
 
-Choose `File` > `New` > `Project...` and select a new Flutter project.
+1. Choose **File>New>Project** (or **Create new project** in the welcome dialog)
+2. In the **New Project Wizard**, select Flutter as your project type (in the left column) and click **Next**
+3. Give your project a name (change location if desired) and click **Finish**
+4. Expand the project view, and double-click `main.dart` inside `lib` to see the main program entrypoint
 
 ### Running an app
 
 Ensure a development device is available and running (a development enabled Android device, or the
 iOS Simulator). In the IntelliJ toolbar, select the launch configuration for the created Flutter 
 project and hit the debug button.
+
+### Additional details
+
+For tips on opening existing projects, debugging, etc. see the [user journeys page](/docs/user_journeys.md).
```

Comment:
Let's land as is, but plan to in-line getting started UI tasks into our user-facing docs, and have the user journeys serve a dual purpose of guiding development efforts, and acting as smoke test scripts for our releases.